rules-tests/PHPOffice/Rector/StaticCall/ChangePdfWriterRector/Fixture/fixture.php.inc

```diff
@@ -20,9 +22,11 @@ namespace Rector\Tests\PHPOffice\Rector\StaticCall\ChangePdfWriterRector\Fixture
 
 final class Fixture
 {
-    public function run(): void
+    public function run(string $format): void
     {
         $writer = new \PhpOffice\PhpSpreadsheet\Writer\Pdf\Mpdf($spreadsheet);
+
+        $writer2 = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($spreadsheet, $format);
```

Comment:
Could you extract it to own fixture file? Now single place is testing 2 different cases

Reply:
Sure. Done